tests/test_skills/test_safe_deployment_abci/test_behaviours.py

```diff
@@ -47,11 +64,192 @@
     BaseSelectKeeperBehaviourTest,
 )
 
-# TOFIX - dep not allowed
-from tests.test_skills.test_oracle_deployment_abci.test_behaviours import (
-    BaseDeployBehaviourTest,
-    BaseValidateBehaviourTest,
-)
+
+class BaseValidateBehaviourTest(FSMBehaviourBaseCase):
+    """Test ValidateSafeBehaviour."""
```

Comment:
So this PR introduces a lot of code repetition, we can open a ticket the refactor, extract and generalise these tests a bit more.